style.css

```diff
@@ -0,0 +1,192 @@
+* {
+  font-family: "Poppins", sans-serif;
+  padding: 0;
+}
+
+.backgroundimg {
+  background-image: url("/image/background.png");
+}
+
+.navmenu {
+  display: flex;
+  justify-content: space-between;
+  margin: 8px 18px;
+  align-items: center;
+  width: 375px;
+  height: 48px;
+  color: #6070ff;
+  font-weight: 700;
+  font-size: 18px;
+}
+
+nav img {
+  margin-right: 50px;
+  display: grid;
+  width: 25px;
+}
+
+.introduction {
+  margin-top: 100px;
+  width: 300px;
+  height: 300px 0 0 0;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  display: flex;
+  align-items: center;
+  color: #172b4d;
+}
+
+.menuSection {
+  flex-direction: column;
+  align-items: center;
+  margin: 8px 20px;
+  display: grid;
+  grid-template-rows: auto auto;
+  padding: 8px 0 0 3px;
+}
+
+.Whoiam {
+  width: 327px;
+  padding-top: 10px;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+}
+
+a {
+  font-style: normal;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+  color: #7f8cff;
+  text-decoration: none;
+}
+
+.socialMedia {
+  width: 170.17px;
+  height: 20px;
+  display: flex;
+  gap: 20px;
+  padding-top: 20px;
+}
+
+a:hover {
+  text-decoration: none;
+}
+
+.cards {
+  display: grid;
+  grid-template-columns: auto;
+}
+
+.card {
+  padding: 15px;
+  margin: 10px 0;
+}
+
+.flex {
+  display: flex;
+  flex-direction: column;
+}
+
+.img {
+  width: 100%;
+}
+
+.topic {
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  color: #172b4d;
+  line-height: 44px;
+  margin-bottom: 10px;
+}
+
+.sub-topic {
+  margin-bottom: 10px;
+}
+
+.sub-topic span {
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #7a869a;
+}
+
+.sub-topic span:first-child {
+  color: #344563;
+}
+
+.dot {
+  height: 5px;
+  width: 5px;
+  background-color: #7a869a;
+  border-radius: 50%;
+  display: inline-block;
+}
+
+.description {
+  line-height: 24px;
+  margin-bottom: 20px;
+  font-size: 15px;
+}
+
+.button-links {
+  margin-bottom: 35px;
+}
+
+ul {
+  list-style-type: none;
+}
+
+.button-links ul {
+  list-style-type: none;
+  display: flex;
+}
+
+.button-links li {
+  align-items: center;
+  padding: 4px 12px;
+  height: 24px;
+  background: #ebebff;
+  border-radius: 8px;
+  margin-right: 5px;
+}
```

Comment:
- [ ] KIndly ensure that you style the color and the background of the technology list to matches the [Figma design template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) chosen 👍.

| Figma Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/63473584/177019139-788fbde7-b310-4e4e-b1bd-28741c56d7b6.png) | ![image](https://user-images.githubusercontent.com/63473584/177019150-eeb583bf-5f5d-412f-a6ee-8b806e1fe7a8.png)  |